test/controllers/beneficial.owner.type.controller.spec.ts

```diff
@@ -12,11 +12,14 @@ import * as config from "../../src/config";
 import { getApplicationData } from '../../src/utils/application.data';
 import {
   BENEFICIAL_OWNER_TYPE_PAGE_HEADING,
+  CHECK_YOUR_ANSWERS_LINK,
+  SERVICE_UNAVAILABLE,
+  TRUST_INFORMATION_LINK
```

Comment:
comma is needed after TRUST_INFORMATION_LINK. if you run 'npm run lint' it should show that - we probably forgot to mention the lint tool is available